src/locale/pt.js

```diff
@@ -46,6 +46,8 @@ export default moment.defineLocale('pt', {
         hh: '%d horas',
         d: 'um dia',
         dd: '%d dias',
+        w: 'uma semana',
+        ww: '%d semanas',
```

Comment:
Hi @caio-ribeiro-pereira, can you verify those two lines?

Reply:
This is okay